src/pages/components/search/accessibility.mdx

```diff
@@ -0,0 +1,259 @@
+---
+title: Search
+tabs: ['Code', 'Usage', 'Style', 'Accessibility']
+---
+
+import {
+  StructuredListWrapper,
+  StructuredListHead,
+  StructuredListBody,
+  StructuredListRow,
+  StructuredListInput,
+  StructuredListCell,
+  OrderedList,
+  ListItem,
+} from 'carbon-components-react';
+
+<PageDescription>
+
+The search Carbon component contains a collection of items and objects that,
+as a whole, combine to create search functionality to content on the website.
+WAI ARIA labels are used on each button so users of assistive technologies
+understand the purpose of the button.
+
+</PageDescription>
```

Comment:
Remove this and put it under 

##Help wanted

Add anchor link for help wanted